.gitmodules

```diff
@@ -14,3 +14,6 @@
 	path = libraries/3rdparty/pkcs11
 	url = https://github.com/amazon-freertos/pkcs11.git
 	branch = v2.40_errata01
+[submodule "libraries/3rdparty/lwip"]
```

Comment:
--name should be "lwip" when doing a git submodule add

Reply:
fixed